lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize()
+    @cells = cells
```

Comment:
No need to set your instance variable here since calling `reset!` will do that for you.